shared/style/action_menu.css

```diff
@@ -280,23 +291,13 @@ html[dir="rtl"] [role="dialog"][data-type="action"] > header > a {
   border-radius: 2rem;
 }
 
-[role="dialog"][data-type="action"] > menu > button:last-child:before {
-  content: '';
-  position: fixed;
-  width: 100%;
-  background-color: #4d4d4d;
-  height: 7rem;
-  offset-inline-start: 0;
-  bottom: 0;
-  z-index: -1;
-}
-
 [role="dialog"][data-type="action"] > menu > button:nth-last-child(2) {
   margin-bottom: 1.5rem;
 }
 
 /* Press state */
-[role="dialog"][data-type="action"] > menu > button:active {
+[role="dialog"][data-type="action"] > menu > button:active,
+[role="dialog"][data-type="action"] > menu > .button-container:last-child button:active {
```

Comment:
if you remove :last-child above, I think you don't need to add the more specific selector here.